src/Annotation/ApiResource.php

```diff
@@ -51,8 +51,9 @@
  *     @Attribute("routePrefix", type="string"),
  *     @Attribute("shortName", type="string"),
  *     @Attribute("subresourceOperations", type="array"),
+ *     @Attribute("sunset", type="string"),
  *     @Attribute("swaggerContext", type="array"),
- *     @Attribute("validationGroups", type="mixed")
+ *     @Attribute("validationGroups", type="mixed"),
```

Comment:
Comma to remove (the Doctrine parser will fail on this IIRC)

Reply:
Should be solved now.